support/checkLinks.js

```diff
@@ -13,8 +13,8 @@ const statAsync = promisify(fs.stat);
 const FILTER_PATTERN = /\.md$/;
 
 /**
- * Need not-robot-like headers to prevent Cloudflare from rejecting our
- * external link checks with 403s.
+ * Use not-robot-like headers to attempt to prevent Cloudflare from
+ * rejecting our external link checks with 403 or 503 status.
```

Comment:
Just to clarify why I'm not deleting this outright. There is still a chance that these headers *might* allow some Cloudflare-hosted links to be checked, because it seems the security levels are configurable on a per-site basis. For example, see the responses here:

- https://stackoverflow.com/a/18523671/2103996
- https://community.cloudflare.com/t/trying-to-curl-to-a-site-protected-by-cloudflare/32918

In short, depends on whether IPs are whitelisted, protections are turned on or off etc.